x-pack/legacy/plugins/siem/public/lib/connectors/resilient/config.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Connector } from '../types';
+
+import { RESILIENT_TITLE } from './translations';
+import logo from './logo.svg';
+
+export const connector: Connector = {
```

Comment:
is this already define in the resilient connector in action? or something similar over there?